allure-cucumber2-jvm/src/main/java/io/qameta/allure/cucumber2jvm/AllureCucumber2Jvm.java

```diff
@@ -50,17 +49,10 @@
 
 import java.io.ByteArrayInputStream;
 import java.nio.charset.Charset;
-import java.util.Deque;
-import java.util.HashMap;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.UUID;
+import java.util.*;
```

Comment:
don't use wildcard imports

Reply:
IntelliJ formatted it. In next commit, it is resolved.